docs/proposed/http-storage-node-protocol.rst

```diff
@@ -380,6 +380,11 @@ then the expiration time of that lease will be changed to 31 days after the time
 If it does not match an existing lease
 then a new lease will be created with this ``renew-secret`` which expires 31 days after the time of this operation.
 
+The renew and cancel secrets must be 32 bytes long
+(or in the case of JSON encoding they must UTF-8 encode to 32 bytes).
```

Comment:
I found this a little confusing on first reading, and then on second reading too.

A 256-bit hash is 32 bytes long—and given it must UTF-8 encode to 32 bytes, how do you fit a full 256-bit hash?

Reply:
Huh... oops!